Gruntfile.js

```diff
@@ -287,6 +287,17 @@ module.exports = function(grunt) {
           return `echo "${version}" > ddocs/medic/version`;
         }
       },
+      setHorticulturalistMetadata: {
+        cmd: () => `
+          mkdir -p ddocs/medic/build_info;
+          cd ddocs/medic/build_info;
+          echo "medic" > application;
+          echo "medic" > namespace;
+          echo "${releaseName}" > version;
+          echo "${new Date().toISOString()}" > time;
+          echo "grunt on \`whoami\`" > author;
+          echo '["medic-api-0.1.0.tgz", "medic-sentinel-0.1.0.tgz"]' > node_modules.json`
+      },
```

Comment:
I think it's nicer to commit the node_modules.json, application, and namespace files rather than generating them every time.

Reply:
Good idea, done!